### ISSUE TITLE
Digito Verificar do Nosso Número não aparece

### ISSUE THREAD

**Issue by leandroportnoy** [BoletoNet/boletonet#55]
Prezados,

tenho gerado a impressão do Boleto do Banco do Brasil e tem saído quase todo correto. Mas o Nosso Número não vem com o digito verificador. Já dei uma pesquisada e não encontro nada sobre... passo o valor também dentro do objeto (tentando forçar) mas o mesmo não vem no Boleto. O que poderia ser?

**Comment by maneskul** [comment]
@leandroportnoy Você está executando o método **Valida()**?

**Comment by leandroportnoy** [comment]
@maneskul , sim! Estruturei meu método no preenchimento das informações e na geração visual do boleto.

        public string MontaHtmlEmbedded()
        {
            boletoBancario.Boleto.Valida();
            return boletoBancario.MontaHtmlEmbedded();
        }

O digito verificador nunca apareceu. É a única pendência que tenho na geração do boleto utilizando essa ferramenta.

**Comment by MarcosFiveDev** [comment]
Alguma coisa quanto a isto? Estou com este mesmo problema para o banco do brasil com carteira 11

**Comment by leandroportnoy** [comment]
Terminei saindo do projeto há pelo menos um ano. Acredito que tenha evoluído algo... se puderem dar retorno ao colega acima...

**Comment by rodolfoghi** [comment]
Fui conferir aqui para ajudar e percebi que de fato na impressão não aparece o dígito verificador. Temos clientes utilizando, e os boletos foram validados pelo banco. Você chegou a enviar para o banco e ele rejeitou?

**Comment by MarcosFiveDev** [comment]
Na verdade o meu cliente está reclamando do digito verificador por conta do setor financeiro dele, visto que outros boletos gerados para o banco do brasil (do mesmo cliente), geram com a exibição do digito.

**Comment by neemiasjunior** [comment]
estou com o mesmo problema aqui... Alguém descobriu qual é o problema?
